addons/payment_authorize/models/authorize_request.py

```diff
@@ -58,9 +58,17 @@ def _make_request(self, operation, data=None):
 
         messages = response.get('messages')
         if messages and messages.get('resultCode') == 'Error':
+            err_msg = messages.get('message')[0].get('text', '')
+
+            # TODO transactionResponse error handling should happen in auth_and_capture() and authorize()
+            tx_errors = response.get('transactionResponse', {}).get('errors')
+            if tx_errors:
+                err_msg += '\n'
```

Comment:
Shouldn't we append `'\n'` only if `err_msg` is not empty?

Reply:
Sure, I added an if.